src/components/GroupList/GroupList.js

```diff
@@ -0,0 +1,33 @@
+/* @flow */
+
+import React, { PropTypes } from 'react'
+import _ from 'lodash'
+import { List } from 'material-ui/List'
+import GroupItem from './GroupItem'
+
+function GroupList(props) {
+  const { data } = props
+  return (
+    <List
+      style={{ paddingTop: 0, paddingBottom: 0 }}
+      primaryTogglesNestedList
+    >
+      {data && data.length &&
+        data.map(item =>
+          <GroupItem
+            key={_.uniqueId('_group_item')}
```

Comment:
Using lodash for generating these ids seems wrong to me... The item should have its own unique identifier already.